speakrune/gen/speakrune.hoon

```diff
@@ -0,0 +1,51 @@
+|=  raw=tape
+^-  tape
+=<  ?:((valid raw) (turn raw convert) "input contains non-rune characters")
+|%
+++  valid
+  |=  x=tape
+  ^-  ?
+  =/  chars  (sy x)
+  (~(all in chars) |=(c=@t (~(has by table) c)))
+++  convert
+  |=  x=@t
+  ^-  @t
+  =/  r  (trip (~(got by table) x))
+  (crip r)
+++  table
+  %-  my
+  :~  :-  ' '  'ace'
+      :-  '|'  'bar'
```

Comment:
Might be cleaner to just do `['|' 'bar']` and similar for all of these. `:-` for a simple, small cons like this is pretty unusual.

Reply:
Makes sense, thanks.